packages/amplify-category-api/resources/awscloudformation/graphql-schemas/basic-schema.graphql.ejs

```diff
@@ -1,7 +0,0 @@
-type <%= props.typeName %> @model {
```

Comment:
Have we removed the base type question? We should probably have an API review for it and separate that out in another PR since it's an independent concern and not tied to this functionality. Thoughts?

Reply:
The problem with pulling it into a separate PR is that it makes this one much more complicated. It's the only schema that requires a value from the walkthrough in order to copy which means to separate the walkthrough and the file i/o would require doing the replacement to a temp file, then copy that to the project and delete the temp file which seems like way more effort than it's worth

Reply:
Okay, let's do a short API review as well as suggested by Richard before we merge this to master.